src/vstart.sh

```diff
@@ -1474,7 +1474,7 @@ do_rgw_create_users()
     local akey='0555b35654ad1656d804'
     local skey='h7GhxuBLTrlhVUyxSPUKUV8r/2EI4ngqJxD7iBdBYLhwluN30JaT3Q=='
     debug echo "setting up user testid"
-    $CEPH_BIN/radosgw-admin user create --uid testid --access-key $akey --secret $skey --display-name 'M. Tester' --email tester@ceph.com -c $conf_fn > /dev/null
+    $CEPH_BIN/radosgw-admin user create --uid testid --access-key $akey --secret $skey --display-name 'M. Tester' --email tester@ceph.com --system -c $conf_fn > /dev/null
```

Comment:
system users are only for rgw multisite, and actually modify the s3 protocol. so this change is breaking our local testing with s3tests

can you please create a different user specifically for the dashboard, and use `--admin` flag instead of `--system`?

Reply:
see https://github.com/ceph/ceph/pull/42339 for the revert

Reply:
> system users are only for rgw multisite, and actually modify the s3 protocol. so this change is breaking our local testing with s3tests
> 
> can you please create a different user specifically for the dashboard, and use `--admin` flag instead of `--system`?

Sorry, I did not know about it. I will create a different user.